src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1288,8 +1320,10 @@ void PolicyHandler::OnActivateApp(uint32_t connection_key,
         application_manager_.connection_handler().get_session_observer());
     permissions.deviceInfo = device_params;
 
-    DeviceConsent consent = policy_manager_->GetUserConsentForDevice(
-        permissions.deviceInfo.device_mac_address);
+    DeviceConsent consent =
+        CallPolicyManagerFunction(&PolicyManager::GetUserConsentForDevice,
+                                  permissions.deviceInfo.device_mac_address);
+
     permissions.isSDLAllowed = kDeviceAllowed == consent;
```

Comment:
@SNiukalov Add round brackets here, please

Reply:
@AByzhynar 
Fixed in: https://github.com/smartdevicelink/sdl_core/pull/2614/commits/98dc17dda0378940c016a6b9a16f1e3992ee8e65